enterprise/cmd/frontend/internal/licensing/enforcement/users.go

```diff
@@ -74,8 +74,7 @@ func NewAfterCreateUserHook() func(context.Context, database.DB, *types.User) er
 			return err
 		}
 
-		// Nil info indicates no license, thus Free tier
-		if info == nil {
+		if info.Plan() == licensing.PlanFree {
```

Comment:
YOOO, how about we add a method like this?

```suggestion
		if info.Plan().IsFree() {
```